.github/workflows/task-get-linux-configurations.yml

```diff
@@ -98,13 +98,42 @@ jobs:
           if x86_platforms.count('amazonlinux:2') > 0:
             x86_include.append({
               'OS': 'amazonlinux:2',
-              'pre-deps': "yum install -y tar gzip"})
+              'pre-deps': "yum install -y tar gzip git"})
+          if 'amazonlinux:2' in arm_platforms:
+            arm_include.append({
+              'OS': 'amazonlinux:2',
+              'pre-deps': "yum install -y tar gzip git"})
 
           # mariner:2 needs pre-checkout dependencies
           if x86_platforms.count('mcr.microsoft.com/cbl-mariner/base/core:2.0') > 0:
             x86_include.append({
               'OS': 'mcr.microsoft.com/cbl-mariner/base/core:2.0',
-              'pre-deps': "tdnf install -y --noplugins --skipsignature tar gzip"})
+              'pre-deps': "tdnf install -y --noplugins --skipsignature tar gzip git ca-certificates"})
+
+            # Ubuntu and Debian distributions need git pre-checkout dependencies
+            debian_based_os = ['ubuntu:noble', 'ubuntu:jammy', 'ubuntu:focal', 'ubuntu:bionic', 'debian:bullseye', 'gcc:12-bookworm']
```

Comment:
This version does not support `ubuntu:noble`, but its name in `debian_based_os` does not cause any problem.